### PR TITLE
Util: Toggle Local vs Published package version

### PR DESCRIPTION
**Pull request by skgbafa**
# Description
This is a utility script for development. It toggles the SSX dependencies from local to the version on NPM and back in all of the packages in the monorepo. This can be tested simply by running
- `node scripts/pre-publish.js`: Replaces `"ssx-": "*"` with  `"ssx-": "x.x.x"`
- `node scripts/pre-publish.js local`:  Replaces `"ssx-": "x.x.x"` with  `"ssx-": "*` 

# Type
- [x] Utility

# Diligence Checklist
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: scripts/toggle-packages.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/**
+ * Command line usage:
+ * `node scripts/toggle-packages.js` // use published package versions
+ * `node scripts/toggle-packages.js` local // use local package versions
```
Comment (Juliano1612) [inline scripts/toggle-packages.js]:
```suggestion
 * `node scripts/toggle-packages.js local` // use local package versions
```

File: scripts/toggle-packages.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        data,
+      };
+    }
+  }
+
+  return packages;
+};
+
+const updatePackageVersion = (packages, usePublishedVersion = true) => {
+  console.log('Using local package versions');
```
Comment (Juliano1612) [inline scripts/toggle-packages.js]:
```suggestion
   console.log(`Using ${usePublishedVersion ? 'remote': 'local'} package versions`);
```

**Review by Juliano1612 — APPROVED** [review]
Nice script! Looks good to me

**Review by w4ll3 — APPROVED** [review]
lgtm

**Pull request merged**